doc/apiv3-documentation.apib

```diff
@@ -2139,6 +2139,84 @@ For more details and all possible responses see the general specification of [Fo
                 "message": "The request body was not empty."
             }
 
+
+# Group Revisions
+
+Revisions are sets of updates to files in the context of repositories linked in OpenProject.
+
+## Linked Properties
+|  Link            | Description                                         | Type          | Constraints | Supported operations |
+|:----------------:| --------------------------------------------------- | ------------- | ----------- | -------------------- |
+| self             | This revision                                       | Revision      | not null    | READ                 |
+| project          | The project to which the revision belongs           | Project       | not null    | READ                 |
+| committer        | The user that committed this revision, if defined   | User          |             | READ                 |
+
+
+## Local Properties
+| Property       | Description                                             | Type        | Constraints | Supported operations |
+|:--------------:| --------------------------------------------------------| ----------- | ----------- | -------------------- |
+| id             | Revisions's id                                          | Integer     | x > 0       | READ                 |
+| identifier     | The SCM identifier of the revision (e.g. SHA hash)      | String      | not null    | READ                 |
+| committerName  | The name of the author that committed this revision     | String      | not null    | READ                 |
```

Comment:
I first thought that this would duplicate the `committer`, but I see that we will probably not be able to match every SCM user to an OpenProject user.

Should we clarify that this is the SCM internal name of the user? (i.e. it can differ from the name the user has in OpenProject)